website/allowed-subcategories

```diff
@@ -62,6 +62,7 @@ Healthcare
 Hybrid Compute
 IoT Central
 IoT Hub
+IoT Operations
```

Comment:
it seems all the documents for resources are missing, could you please add documents for them